src/views/EditSidebar.vue

```diff
@@ -221,8 +221,10 @@
 			</template>
 			<div class="app-sidebar-tab__content">
 				<InviteesList v-if="!isLoading"
+					:calendar-instance="selectedCalendar"
```

Comment:
```suggestion
					:calendar="selectedCalendar"
```

Reply:
Just personal preference and consistency in the code base. Your call ... (also applies to other reviews regarding `calendar-instance`).